keg_auth/testing.py

```diff
@@ -87,15 +95,19 @@ def test_login_field_success_next_session(self):
             client = flask_webtest.TestApp(flask.current_app)
             with client.session_transaction() as sess:
                 sess['next'] = next
-            resp = client.get(self.login_url)
+            resp = client.get(flask.url_for(flask.current_app.auth_manager.endpoint('login')))
 
             resp.form['login_id'] = 'foo@bar.com'
             resp.form['password'] = 'pass'
             resp = resp.form.submit()
 
         assert resp.status_code == 302, resp.html
         assert resp.headers['Location'] == 'http://keg.example.com{}'.format(next)
-        assert resp.flashes == [('success', 'Login successful.')]
+        flash_success = flask.current_app.auth_manager.login_authenticator_cls.\
+            responder_cls['login'].flash_success
```

Comment:
In the tests its find to use `# noqa` to get around line length limits in Flake 8 This is a good example of where it is probably cleaner to do that.